compiler/aarch64/codegen/BinaryEvaluator.cpp

```diff
@@ -30,6 +30,67 @@
 #include "il/Node_inlines.hpp"
 #include "infra/Bit.hpp"
 
+/**
+ * Generic helper that handles a number of similar binary operations.
+ * @param[in] node : calling node
+ * @param[in] regOp : the target AArch64 instruction opcode
+ * @param[in] regOp : the matching AArch64 immediate instruction opcode
```

Comment:
regOpImm

Reply:
Also, please document here the assumption from below that regOp == regOpImm implies there isn't an immediate form instruction.